src/main/java/uk/gov/hmcts/divorce/divorcecase/model/GeneralEmail.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.divorce.divorcecase.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.ccd.sdk.api.CCD;
+import uk.gov.hmcts.divorce.divorcecase.model.access.CaseworkerAccessBetaOnlyAccess;
+
+import static uk.gov.hmcts.ccd.sdk.type.FieldType.Email;
+import static uk.gov.hmcts.ccd.sdk.type.FieldType.FixedList;
+import static uk.gov.hmcts.ccd.sdk.type.FieldType.TextArea;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class GeneralEmail {
+
+    @CCD(
+        label = "Address to",
+        access = {CaseworkerAccessBetaOnlyAccess.class},
```

Comment:
You could either set the access on the property in CaseData or not set it at all and rely on the permissions being picked up from the event permissions

Reply:
didn't realised that will remove it from here and will rely on event permissions.